addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -285,6 +296,7 @@ const UserValueWidget = Widget.extend({
         this._methodsNames = [];
         this._methodsParams = _.extend({}, extraParams);
         this._methodsParams.optionsPossibleValues = {};
+        this._methodsParams.widgetPossibleValues = [];
```

Comment:
Not sure about this key, seems ultra specific to your widget only? Should not it be named `record` and returned only through `params` when using that particular widget? And it should be a map widgetValue -> value instead of an array.

Reply:
I still need those params to be shared to the parent Snippet as my m2o is in a select. That's why I made it more generic.